README.md

```diff
@@ -0,0 +1,59 @@
+# Introduction
+
+Physical Quantity is a single-header c++ library providing compile-time SI unit consistency.
+
+Physical Quantity implements most of the International System of Units (SI),
+with a few exceptions (angles, for example).  Once you start using it, your
+compiler will take care of making sure all units are converted properly, at
+compile time.
+
+Here's an example:
+
+```
+  // When I am in a hurry, I walk
+  Length<double> length = Length<double>::kilometer(1.0);
+  // in
+  Duration<double> time = Duration<double>::minute(10);
+
+  // Then my speed is
+  Velocity<double> velC = length / time;
+
+  std::cout << "My walking speed: " << velC.metersPerSecond() << "m/s, "
```

Comment:
This is a detail, but it seems like the convention in the code is to name all units in *singular*. So to be consistent with that, it would be "meterPerSecond" (not "meter**s**PerSecond"), the same goes for "kilometerPerHour" and "knot"

Reply:
that was the case in the code, but not in the doc. Thanks for spotting the mistake.